inc/customizer/base_customizer.php

```diff
@@ -390,30 +390,28 @@ public function add_boxed_layout_controls( $id, $settings ) {
 			];
 		}
 
-		if ( neve_is_new_skin() ) {
-			$padding_live_refresh_settings = [
-				'cssVar' => array(
-					'vars'       => '--padding',
-					'selector'   => $settings['boxed_selector'],
-					'responsive' => true,
-				),
-			];
+		$padding_live_refresh_settings = [
+			'cssVar' => array(
+				'vars'       => '--padding',
+				'selector'   => $settings['boxed_selector'],
+				'responsive' => true,
+			),
+		];
+
+		$background_live_refresh_settings = [
+			'cssVar' => array(
+				'vars'     => '--bgcolor',
+				'selector' => $settings['boxed_selector'],
+			),
+		];
 
-			$background_live_refresh_settings = [
+		if ( $has_text_color ) {
+			$color_live_refresh_settings = [
 				'cssVar' => array(
-					'vars'     => '--bgcolor',
+					'vars'     => '--color',
 					'selector' => $settings['boxed_selector'],
 				),
 			];
```

Comment:
we can move L409-414 to inside of the first `if ( $has_text_color ) {` check. So, I think we should combine `if ( $has_text_color ) { `calls

also as far I can see, `$color_live_refresh_settings = ['template' => $template,];` is overridden by the second

<img width="1188" alt="Screen Shot 2023-02-18 at 03 01 56" src="https://user-images.githubusercontent.com/25361626/219818996-6d1f24ea-0674-4306-a1c1-d957b44e5516.png">